common/nymcoconut/src/proofs/mod.rs

```diff
@@ -88,23 +86,21 @@ impl ProofCmCs {
     /// using the Fiat-Shamir heuristic.
```

Comment:
Just a sanity check, since I have absolutely no idea, does this comment still hold true?

Reply:
Yes, we still use the Fiat-Shamir heuristic in our zk proofs